### PR TITLE
feat: add checklist variant to list

### PR DESCRIPTION
**Pull request by toniQ97**
# Description

Added variant for list component to render items as a checklist

# Screenshots
### Desktop
image
### Mobile 
image

# Checklist

Pull request contains:

- [ ] A new component
- [X] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [X] Includes theme changes for both Uswitch and money
- [X] Work has been tested in multiple browsers
- [X] PR description includes description of change
- [X] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/themes/broadband-deals/theme.json [hunk 1]
```diff
@@ -261,10 +261,16 @@
         }
       },
       "pros": {
-        "color": "#22ACFF"
+        "color": "#22ACFF",
```
Comment (phelma) [inline src/themes/broadband-deals/theme.json:264]: lets give this a name and put it in colors

File: src/themes/broadband-deals/theme.json [hunk 2]
```diff
@@ -310,6 +316,31 @@
...
+                "content": "none"
+              },
+              "svg": {
+                "display": "inline",
+                "mr": ["sm", "md"],
+                "verticalAlign": "middle"
+              }
+            },
+            "border": "1px solid",
+            "borderColor": "rgba(104,104,135,0.3)",
```
Comment (phelma) [inline src/themes/broadband-deals/theme.json:333]: this one too

File: src/elements/list/src/stories.tsx [hunk 3]
```diff
@@ -5,13 +5,19 @@ import { jsx } from 'theme-ui'
 import AllThemes from '../../../utils/all-themes'
 
 import { List, ListItem } from './'
+import { BBDealsExample } from '../../hero/src/stories'
```
Comment (phelma) [inline src/elements/list/src/stories.tsx:8]: I don't think that this is used

Comment (phelma) [reply src/elements/list/src/stories.tsx:8]: this isn't being used and is failing the CI

Comment (toniQ97) [reply src/elements/list/src/stories.tsx:8]: Sorry, auto import
I'll fix that in a minute

**Pull request merged**